bootloader/grub.go

```diff
@@ -370,5 +376,96 @@ func (g *grub) ManagedAssets() []string {
 //
 // Implements ManagedAssetsBootloader for the grub bootloader.
 func (g *grub) CommandLine(extra []string) (string, error) {
-	return "", fmt.Errorf("not implemented")
+	// we do not trust the on disk asset, use the built-in one
+	assetName := "grub.cfg"
+	if g.recovery {
+		assetName = "grub-recovery.cfg"
+	}
+	staticCmdline, err := staticCommandLineFromGrubAsset(assetName)
+	if err != nil {
+		return "", fmt.Errorf("cannot extract static command line element: %v", err)
+	}
+	bv, err := g.GetBootVars("snapd_extra_cmdline_args")
```

Comment:
This is incorrect for a particular recovery system, as we're fetching from the grubenv in the system-seed, while we should be fetching from grubenv inside `system-seed/systems/<label>`